### PR TITLE
Updating frontend on server-testing

### PR DESCRIPTION
**Pull request by umee007**
This pull request is to move up to date frontend in server testing for implementing backend logic, and resolve any possible conflicts in code, folder structure and imports etc.

### CODE REVIEW

**Review by umee007 — COMMENTED** [review]
Changes reviewed and noted to avoid and manage conflicts in similar code (to implement server side's backend logics on frontend code).

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import Login from "./pages/Login";
+import Login from "./pages/login/Login";
```
Comment (tombstone-10) [inline client/src/App.jsx:1]: create login folder in pages and move Login component there.

File: client/src/components/Tabs.jsx [hunk 2]
```diff
@@ -1,39 +1,19 @@
...
-      icon: <FaLaptop />,
-    },
-    {
-      path: "/profile/security",
-      name: "Security",
-      icon: <FaShieldAlt />,
-    },
-    {
-      path: "/profile/about",
-      name: "About",
```
Comment (tombstone-10) [inline client/src/components/Tabs.jsx:20]: these will be removed as there is a new folder data, containing data file tabsData.jsx. from there data is passed as props for dynamic rendering based on the page it is called in.

**Review by tombstone-10 — APPROVED** [review]

**Comment by tombstone-10** [comment]
Opening a new pr from github desktop.

**Pull request closed without merging**